powerspikegg/frontend/src/app/services/gateway.service.ts

```diff
@@ -73,6 +76,12 @@ export class GatewayService {
       .catch(error => Observable.throw(error));
   }
 
-  // TODO(ArchangelX360): computation request
+  getComputedStatistics(query: ComputationQuery): Observable<MatchComputationFeature> {
+    const url = COMPUTATION_API_URL + '/' + query.summonerID + '/' + query.matchID + '/' + query.region;
+    console.log('Calling: ' + url);
```

Comment:
I think Angular CLI prod aot compilation removes console.log so it is not a problem and stays in debug mode only anyway.